gitbook-docs/market/front-end.md

```diff
@@ -1,137 +1,124 @@
 ---
-description: Proof Market front end guide
+description: Web interface of the Proof Market
 ---
 
-# Front End
+# Web interface
+
+The web interface of the Proof Market can be accessed at [**proof.market**](https://proof.market).
 
 {% hint style="info" %}
 Please note this product is in beta and expected to change.
 {% endhint %}
 
-#### Proof Market front end can be accessed at [**proof.market**](https://proof.market)
-
-## New user signup
-
-All users who wish to interact with the proof market must create an account.
-
-{% hint style="info" %}
-If you do not have the credentials, users can still view the exchange, but they will be unable to post orders to buy/sell proofs.
-{% endhint %}
+## Creating an account 
 
-<figure><img src="../.gitbook/assets/image (6).png" alt=""><figcaption></figcaption></figure>
+To interact with the Proof Market, create an account at [proof.market](https://proof.market)
+or [through the command-line interface](cmd-reference/user.md).
+Without an account, you can view the exchange, but you won't be able to post orders 
+to buy or sell proofs.
 
-Click on Sign up
+To register a new user via the web interface, click **Sign in** from the main page
+and then click **Sign up**:
 
-<figure><img src="../.gitbook/assets/image.png" alt=""><figcaption></figcaption></figure>
+<figure><img src="../.gitbook/assets/sign-in.png" alt=""><figcaption><p>Sign in view</p></figcaption></figure>
 
-Select your username and password and complete the flow by clicking register.
+Or you can go directly to the [register page](https://proof.market/#/register).
```

Comment:
This is easier than clicking through the "Sign in" screen. Let's put it before the picture.

To make an account on the Proof Market, go to the [Sign up](https://proof.market/#/register) page or select **Sign in** > **Sign up** on the main page.